File: PERL/Cap02/c02ex01.pl
# c02ex01.pl

#!/usr/bin/perl

use strict;
use warnings;

# Operadores binários

print 5 + 3 . "\n";       # Saída: 8
print 10 - 4 . "\n";      # Saída: 6
print 2 * 6 . "\n";       # Saída: 12
print 10 / 3 . "\n";      # Saída: 3.3333... (quociente real)
print int(10 / 3) . "\n"; # Saída: 3 (quociente inteiro)
print 10 % 3 . "\n";      # Saída: 1 (resto da divisão)

# Operadores unários

print +5 . "\n";          # Saída: 5 (mantém sinal po-sit.)
print -8 . "\n";          # Saída: -8 (inverte sinal ne-gat.)

# Exponenciação

print 2 ** 3 . "\n";      # Saída: 8 (2 elevado a 3)

File: RUST/Cap02/c02ex01.rs
// c02ex01.rs

fn main() {
  // Operadores binários
  println!("{}", 5 + 3);       // Saída: 8
  println!("{}", 10 - 4);      // Saída: 6
  println!("{}", 2 * 6);       // Saída: 12
  println!("{}", 10.0 / 3.0);  // Saída: 3.3333... (quociente real)
  println!("{}", 10 / 3);      // Saída: 3 (quociente inteiro)
  println!("{}", 10 % 3);      // Saída: 1 (resto da divisão)

  // Operador unário
  println!("{}", -8);          // Saída: -8 (inverte sinal)

  // Exponenciação
  println!("{}", f64::powf(2.0, 3.0)); // Saída: 8 (2 elevado a 3)
}
